src/components/SearchProducts/SearchProductCard/SearchProductCard.tsx

```diff
@@ -0,0 +1,30 @@
+import { Image, View } from "react-native";
+import { Typography } from "../../Typography";
+import { styles } from "./styles";
+import { Product } from "../../ProductSlide";
+
+interface Props {
+  product: Product,
+  position: number
```

Comment:
Why not just use Flexbox or Grid here?? using `nth-child(odd)` or something like this?